pkg/ingester/ingester.go

```diff
@@ -167,6 +176,10 @@ type Ingester struct {
 
 	// Prometheus block storage
 	TSDBState TSDBState
+
+	// Rate of pushed samples. Only used by V2-ingester to limit global samples push rate.
+	ingestionRate        *ewmaRate
+	inflightPushRequests atomic.Int64
```

Comment:
Are alignment issues like this worth considering (seems to affect uber-go/atomic as well)?

https://github.com/grafana/loki/commit/2dbdbee5e67999b255ec25f510b993792b8d46ee

https://github.com/golang/go/issues/36606

Reply:
Mentioned problem only affects 32-bit systems. I don't think we have any policy in Cortex, but running Cortex on 32-bit system seems like bad idea to me, so I would ignore this problem. (It should really be solved at Go-level, we can only work around it here.)